docs/_static/f5-openstack-agent.gre.ini

```diff
@@ -97,8 +97,6 @@ f5_ha_type = standalone
 #
 # Sync mode
```

Comment:
This config setting has been completely removed from the ini file.  All doc references, config examples, etc. should also remove the section.